python/looker/rtl/api_methods.py

```diff
@@ -33,7 +34,7 @@ def get(self,
             structure: sr.TStructure,
             path: str,
             query_params: Optional[MutableMapping[str, str]] = None
-            ) -> sr.TDeserializeReturn:
+           ) -> sr.TDeserializeReturn:
```

Comment:
funny, our separate usages of yapf produce different results! might be this issue ["YAPF would at times not produce deterministic formatting..."](https://news.ycombinator.com/item?id=17155048) I'm curious to try out [Black](https://github.com/psf/black)